packages/patterns/src/components/pattern-overrides-controls.js

```diff
@@ -92,6 +92,7 @@ function PatternOverridesControls( {
 		<>
 			<InspectorControls group="advanced">
 				<BaseControl
+					__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the editor, add an Image block. From the block toolbar three-dot menu, choose "Create pattern". Click "Edit original". In the block inspector, expanded the Advanced section to see the Enable overrides button at the bottom.

<img src="https://github.com/user-attachments/assets/c45eb31c-6f8c-493b-b527-fc15ffa3fb97" alt="Edit pattern original block inspector" width="273">